### PR TITLE
edited webserver config template for dnsdist docker container.

### PR DESCRIPTION
**Pull request by Houtworm**
### Short description

Edited a single line which contais the webserver config template for dnsdist.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: dockerdata/startup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 elif product == 'dnsdist':
     args = ['--supervised', '--disable-syslog']
     apienvvar = 'DNSDIST_API_KEY'
-    apiconftemplate = """webserver("0.0.0.0:8083", '{{ apikey }}', '{{ apikey }}', {}, '0.0.0.0/0')
+    apiconftemplate = """webserver("0.0.0.0:8083")
```
Comment (rgacogne) [inline dockerdata/startup.py:39]: I think we still want to set the API key, password and ACL like it was done before, so:
```suggestion
    apiconftemplate = """webserver("0.0.0.0:8083")
    setWebserverConfig({password='{{ apikey }}', apiKey='{{ apikey }}', acl='0.0.0.0/0'})
```

**Review by rgacogne — CHANGES_REQUESTED** [review]
Thanks for this pull request, much appreciated!

**Review by Houtworm — COMMENTED** [review]
I don't know if this "review" from my side is needed, But better safe than sorry :P

Thanks for the changes, I was not sure where to put them. It seems to pass the checks too :)

**Comment by Houtworm** [comment]
It works now :)

**Review by rgacogne — APPROVED** [review]

**Pull request merged**